unix_env_setup.sh

```diff
@@ -2,7 +2,7 @@
 
 sudo apt-add-repository -y "ppa:ubuntu-toolchain-r/test"
 sudo apt-get update
-sudo apt-get install g++-8 unzip autoconf libtool libssl-dev python perl libncurses5-dev libreadline-dev
+sudo apt-get install g++-8 unzip autoconf libtool libssl-dev python libncurses5-dev libreadline-dev
```

Comment:
g++-9 instead of g++-8 no?

Reply:
no, I maintain this to match what we are using on travis. Travis uses g++-8